### PR TITLE
DOCS-4099 - unattended windows installation instructions

### PR DESCRIPTION
**Pull request by mpaik**

### CODE REVIEW

File: source/includes/fact-windows-unattended.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Install Windows Unattended
```
Comment (tychoish) [inline source/includes/fact-windows-unattended.rst]: unclear to me what "unattended" means?

File: source/includes/fact-windows-unattended.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+   msiexec.exe /q /i mongodb-<version>-signed.msi INSTALLLOCATION="<installation directory>"
+
+By default, this method installs all MongoDB binaries except the router (``mongos.exe``) and miscellaneous tools (``bsondump.exe``, ``mongofiles.exe``, ``mongooplog.exe``, and ``mongoperf.exe``) to the directory specified in ``<installation directory>``. To install specific subsets of the binaries, you may specify an ``ADDLOCAL`` argument:
+
+.. code-block:: powershell
+
+   msiexec.exe /q /i mongodb-<version>-signed.msi INSTALLLOCATION="<installation directory>" ADDLOCAL=<binary set(s)>
+
+where ``<binary set(s)>`` is a comma-separated list of one or more of:
```
Comment (tychoish) [inline source/includes/fact-windows-unattended.rst]: should be a sentence

File: source/includes/fact-windows-unattended.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+You may install MongoDB unattended on Windows from the command line
+through the use of ``msiexec.exe``. Open a shell in the directory
+containing the ``.msi`` installation binary of your choice and
+invoke:
+
+.. code-block:: powershell
+
+   msiexec.exe /q /i mongodb-<version>-signed.msi INSTALLLOCATION="<installation directory>"
+
+By default, this method installs all MongoDB binaries except the router (``mongos.exe``) and miscellaneous tools (``bsondump.exe``, ``mongofiles.exe``, ``mongooplog.exe``, and ``mongoperf.exe``) to the directory specified in ``<installation directory>``. To install specific subsets of the binaries, you may specify an ``ADDLOCAL`` argument:
```
Comment (tychoish) [inline source/includes/fact-windows-unattended.rst]: line length.

File: source/includes/fact-windows-unattended.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
+Install Windows Unattended
+--------------------------
+
+You may install MongoDB unattended on Windows from the command line
+through the use of ``msiexec.exe``. Open a shell in the directory
```
Comment (tychoish) [inline source/includes/fact-windows-unattended.rst]: through use of > using

File: source/includes/fact-windows-unattended.rst (hunk 2 above)
Comment (mpaik) [inline source/includes/fact-windows-unattended.rst]: Disagree.

File: source/includes/fact-windows-unattended.rst (hunk 1 above)
Comment (mpaik) [inline source/includes/fact-windows-unattended.rst]: Should be clear to Windows users; commonplace terminology in the Windows ecosystem.

File: source/includes/fact-windows-unattended.rst (hunk 3 above)
Comment (mpaik) [inline source/includes/fact-windows-unattended.rst]: Fixed.

File: source/includes/fact-windows-unattended.rst (hunk 4 above)
Comment (mpaik) [inline source/includes/fact-windows-unattended.rst]: Fixed.

**Comment by tychoish** [comment]
merged

**Pull request closed without merging**